### PR TITLE
[Home] Show artist rails.

### PR DESCRIPTION
**Pull request by alloy**
@sarahscott @mennenia Be sure to read updated instructions for the configuration file!

### CODE REVIEW

File: Example/Emission/Configuration.h.sample [hunk 1]
```diff
@@ -1,8 +1,19 @@
...
-// $ curl -X POST "https://api.artsy.net/oauth2/access_token?client_id=e750db60ac506978fc70&client_secret=3a33d2085cbd1176153f99781bbce7c6&grant_type=credentials&scope=offline_access&email=...&password=..."
+//   $ curl -s -X POST "https://api.artsy.net/oauth2/access_token?client_id=e750db60ac506978fc70&client_secret=3a33d2085cbd1176153f99781bbce7c6&grant_type=credentials&scope=offline_access&email=[EMAIL]&password=[PASSWORD]" | ruby -r json -e 'puts JSON.parse(ARGF.read)["access_token"]' | tee /dev/tty | pbcopy
+//
+// The token will also be placed on your pasteboard, so you can directly paste it below in the `OAUTH_TOKEN` macro.
+//
+// You will also need to get your user ID, which you can do with the following command, using the token you just retrieved:
+//
+//   $ curl -s -H 'x-access-token: [OAUTH_TOKEN]' "https://api.artsy.net/api/v1/me" | ruby -r json -e 'puts JSON.parse(ARGF.read)["_id"]' | tee /dev/tty | pbcopy
+//
+// Again, your user ID will be placed on your pasteboard, so you can directly paste it below in the `USER_ID` macro.
```
Comment (mennenia) [inline Example/Emission/Configuration.h.sample]: 🙇 such wonderful instructions

File: lib/components/home/artist_rail.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react';
+import { View, Text } from 'react-native';
+
+import Spinner from '../spinner';
+
+class ArtistRail extends React.Component {
```
Comment (mennenia) [inline lib/components/home/artist_rail.js]: Thanks for the example file! Is it ok if I merge this with the files I already have in my branch shortly?

**Comment by mennenia** [comment]
This looks great!!

File: lib/components/home/artist_rail.js (hunk 2 above)
Comment (alloy) [inline lib/components/home/artist_rail.js]: Bien sûr!

**Pull request merged**